src/Components/Common/components/PatientDetail.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from "react";
+import { classNames } from "../../../Utils/utils";
+
+export const PatientDetail = ({
+    name,
+    children,
+    className,
+  }: {
+    name: string;
+    children?: ReactNode;
+    className?: string;
+  }) => {
+    return (
+      <div
+        className={classNames(
+          "inline-flex items-center whitespace-nowrap text-sm tracking-wide",
+          className,
+        )}
+      >
+        <div className="font-medium text-secondary-800">{name}: </div>
+        {children != null ? (
+          <span className="pl-2 font-bold">{children}</span>
+        ) : (
+          <div className="h-5 w-48 animate-pulse bg-secondary-200" />
+        )}
+      </div>
+    );
+  };
+  
```

Comment:
1. Is there a reason for this component to be named specific to "Patient"? It accepts any key value pair right? Consider renaming the component to something more appropriate.
2. Why is this a named export? This file only exports this one component right? Prefer default export.

Reply:
is KeyValueDetail appropriate name for this component ?

Reply:
`DeferredDetail` or `AsyncDetail` would make better sense as the primary goal of this component is to show a loading state when children is null right?


Just my thoughts (feel free to do whichever you prefer):

There are many other places where "key/value" like information is represented similarly in different ways (such as a common shared loading state, rather than individual attribute level loading state and so on).

I would prefer to keep this component defined locally in both the Prescriptions print and Treatment Summary print components. This would give us better control over the design as per context. Making it generic and reusable for such a small piece is not required.